File: Weatley.Web/src/app/core/entities/accounting.ts
import { Customer } from './customer';

export class Accounting {
	id: string;
	finalPrice: number;
	date: Date;
	paymentType: string;
	customer: Customer;
	paid: boolean;

	constructor(values: Object = {}) {
		Object.assign(this, values);
	}
}

File: Weatley.Backend/Weatley.Model/Entities/Accounting.cs
using System;
using System.Collections.Generic;
using System.Text;
using Weatley.Model.Enums;

namespace Weatley.Model.Entities
{
    public class Accounting : IEntityBase
    {
        public Guid Id { get; set; }
        public double FinalPrice { get; set; }
        public DateTime Date { get; set; }
        public PaymentTypeEnum PaymentType { get; set; }
        public Customer Customer { get; set; }

    }
}
